dashboard/app/helpers/lti_helper.rb

```diff
@@ -0,0 +1,110 @@
+
+module LtiHelper
+
+  #
+  # LTI role can be blank, one, or multiple (comma separated).
+  # https://www.imsglobal.org/specs/ltiv1p0/implementation-guide#toc-19
+  #
+  # e.g. "Instructor, TeachingAssistant, ContentDeveloper"
+  #
+  def lti_role_to_user_type(roles)
+    return User::TYPE_STUDENT if roles.blank?
+
+    arr = roles.downcase.split(',').uniq.compact
```

Comment:
```
irb(main):001:0> roles = "Instructor, TeachingAssistant, ContentDeveloper"
=> "Instructor, TeachingAssistant, ContentDeveloper"
irb(main):002:0> roles.downcase.split(',').uniq.compact
=> ["instructor", " teachingassistant", " contentdeveloper"]
```
This preserves spaces in the array, do you mean to split on a regex like `/\s+,\s+/` instead?